src/jdk.jpackage/share/native/applauncher/JvmLauncher.cpp

```diff
@@ -33,6 +33,11 @@
 #include "Toolbox.h"
 #include "ErrorHandling.h"
 
+#if defined(_WIN32) && !defined(_WIN64)
+#define LAUNCH_FUNC "_JLI_Launch@56"
```

Comment:
Why JLI_Launch does not work for 32-bit? Do you know why @56 is being used?

Reply:
not so much that is doesn't work, but that it doesn't exist,
The 32 bit version of jli.dll does not have a "JLI_LAUNCH" entry point, but does have a "_JLI_Launch@56" which works perfectly well.